layers/meta-resin-genericx86/recipes-kernel/linux/linux-yocto_4.4.bbappend

```diff
@@ -0,0 +1,30 @@
+inherit kernel-resin
```

Comment:
there already is a `linux-yocto_%.bbappend` you can use to add these into. Please us that one

Reply:
This patches are specific to linux-yocto version 4.4

Reply:
The kernel version we currently have on Intel NUC is 4.12.12
You suggest we downgrade the kernel to 4.4?